isbg.py

```diff
@@ -299,7 +299,7 @@ def dehexof(x):
         imapport = 993
 
 if pastuidsfile is None:
-    pastuidsfile = os.path.expanduser("~" + os.sep + ".isbg-track")
+    pastuidsfile = os.path.expanduser("~" + os.sep + ".isbg-track" + "%" + imapuser + "%" + imapproxyuser)
```

Comment:
I thought about chars to use for separators - a percent is less likely in a user (or account) name than a dash, and the latter might lead to ambiguous conflicts, e.g. `.isbg-track-smith-wesson-admin` : which is the user and the admin part? what if your domain has both `smith`, `smith-wesson`, `wesson-admin` and `admin` accounts, and they happen to be active for isbg at the same time?

Possibly a contrived example, but when things like these run for a server with thousands of boxes, the law of big numbers kicks in and causes problems with non-negligible probability :) An improbable username character does away with this.